src/components/PreviewFrame.jsx

```diff
@@ -100,6 +110,13 @@ class PreviewFrame extends React.Component {
     });
   }
 
+  _postHighlighterSelectorToFrame(selector) {
+    window.frames[0].postMessage(JSON.stringify({
+      type: 'highlightElement',
```

Comment:
I think we should stick with the convention established for posting errors e.g. use `org.popcode.highlightElement`

Reply:
changed to org.popcode.highlightElement